docs/src/user_docs/assembly/extension_field_operations.md

```diff
@@ -0,0 +1,15 @@
+# Extension Field Operations
+Miden assembly provides a set of instructions which can perform operations with extension field elements. These instructions are described in the table below.
+
+### Arithmetic operations
+
+| Instruction | Stack Input | Stack Output | Notes |
+| ----------- | ----------- | ------------ | ----------- |
+| ext2_add <br> - *(5 cycles)*  <br>  | [b, a, ...] | [c, ...] | $c \leftarrow (a + b) \mod p$                                |
+| ext2_sub <br> - *(8 cycles)*  <br>  | [b, a, ...] | [c, ...] | $c \leftarrow (a - b) \mod p$                                |
+| ext2_mul <br> - *(24 cycles)*  <br> | [b, a, ...] | [c, ...] | $c \leftarrow (a \cdot b) \mod p$                            |
+| ext2_div <br> - *(59 cycles)*  <br> | [b, a, ...] | [c, ...] | $c \leftarrow (a \cdot b^{-1}) \mod p$ <br> Fails if $b = 0$ |
+| ext2_neg <br> - *(4 cycles)*  <br>  | [a, ...]    | [b, ...] | $b \leftarrow -a \mod p$                                     |
+| ext2_inv <br> - *(33 cycles)*  <br> | [a, ...]    | [b, ...] | $b \leftarrow a^{-1} \mod p$ <br> Fails if $a = 0$           |
+
+where p = irreducible polynomial x^2 - x + 2 over F_q, q = 2^64 - 2^32 + 1
```

Comment:
@itzmeanjan I think this is where it is mentioned

Reply:
@Al-Kindi-0 I actually mentioned it after @itzmeanjan's comment. It wasn't there before.

Reply:
nit: I would use math notation here. Maybe something like:

where $p$ is an irreducible polynomial $x^2 - x + 2$ over $F_q$ for $q = 2^{64} - 2^{32} + 1$.